generators/template/templates/template.hbs

```diff
@@ -6,8 +6,10 @@ template: true
 <!-- template:<%= name %> -->
 <% if (tag != "div") { %><%- "{{#parseFixture '" + name + ".json'}}\n<" + tag + " class=\"" + name + "\">\n    " + pretty + "\n</" + tag + ">\n{{/parseFixture}}" %>
 <% } else { %>{{#parseFixture '<%= name %>.json'}}
-<div class="<%= name %>">
-	<%= pretty %>
-</div>
+<main id="main" class="main">
+	<div class="<%= name %>">
```

Comment:
Probably can just replace the div with the main you added, instead of having both.

Also wanna make sure that the access nav in the assemble-structure repo refers to this ID.

Reply:
I already updated the access nav to include #main.

I'll remove the div.